src/dotnet-install.ps1

```diff
@@ -1012,24 +1012,24 @@ function Install-Dotnet {
             Say "NOTE: Due to finding a version manifest with this runtime, it would actually install with version '$EffectiveVersion'"
         }
 
-        return
+        exit 0
```

Comment:
exiting out of function makes the code really hard to read and maintain. Is it a regular practice to do so in PowerShell?